frame/test_screencopy_bandwidth.py

```diff
@@ -57,3 +59,25 @@ async def test_inactive_app(self, record_property, server, app) -> None:
         async with server as s, tracker, s.program(app):
             await asyncio.sleep(long_wait_time)
         _record_properties(record_property, server, tracker, 2)
+
+    async def test_app_dragged_around(self, record_property, server) -> None:
+        async def pause():
+            await asyncio.sleep(0.2)
+        extensions = ScreencopyTracker.required_extensions + VirtualPointer.required_extensions
+        app_path = Path(__file__).parent / 'clients' / 'maximizing_gtk_app.py'
+        server = DisplayServer(server, add_extensions=extensions)
+        app = server.program(('python3', str(app_path)))
+        tracker = ScreencopyTracker(server.display_name)
+        pointer = VirtualPointer(server.display_name)
+        async with server, tracker, app, pointer:
+            await asyncio.sleep(1)
+            pointer.move_to_proportional(0.5, 0.03)
```

Comment:
This didn't work on my 2256×1504 display when native. Lab will run 1920×1080, so if that works, we'll be OK.